client/request_handler.go

```diff
@@ -23,13 +23,12 @@ func NewRequestHandler(client BaseClient) *RequestHandler {
 }
 
 func (c *RequestHandler) sendRequest(method string, rawURL string, data url.Values,
-	headers map[string]interface{}) (*http.Response, error) {
+	headers map[string]interface{}, body ...byte) (*http.Response, error) {
```

Comment:
In case of json body will data be null ? If yes, then try to check if method overload possible in go ?

Reply:
Method overloading is not possible in go, in case of json body data need to be null, it can contain query parameters